lib/frisby.js

```diff
@@ -1217,7 +1217,7 @@ function _jsonContains(jsonBody, jsonTest) {
         var keyType = jsonTest[key].prototype;
 
         // User-supplied callback (anonymous function)
-        if(_toType(keyType) === "object" && arguments.callee.caller.name === "") {
+        if(_toType(keyType) === "object" && (arguments.callee.caller.name === "" || arguments.callee.caller.name === arguments.callee.name)) {
```

Comment:
the `argument.callee...` part is removed in master; I'm not sure but it looks that it should be enough.